### ISSUE TITLE
Game controller issue with multiple Godot game instances.

### ISSUE THREAD

**Issue by capnm** [godotengine/godot#33656]
**Godot version:**
98caeb635c2e4d48e0d6d6dbd82af183ab83f894
Date:   Fri Nov 15 13:31:32 2019 +0100
**OS/device including version:**
Linux, Ubuntu 18.04.3 LTS
**Issue description:**
While preparing a multiplayer demo (by the way, Godot's hi-level network works much better than bevore), one kid brought an original PS4 controller and I noticed that Godotengine is receiving the controller (USB) events in parallel in all started Game instances, regardless of whether the window is selected or not. I expected it to behave like other input devices.

In the practice, this is only a problem by multiplayer development on a single computer.
I also could not find a GDScript function that indicates whether the application window is selected or not.

**Comment by agameraaron** [comment]
+1
Yes, please only allow gamepad input when window is in focus. 
Even with one instance this is an issue for me.

**Comment by Calinou** [comment]
Duplicate of #16832.

**Comment by capnm** [comment]
Thanks! I missed the constants in
https://docs.godotengine.org/en/latest/classes/class_mainloop.html :-)
With the notification event, I could track the focus state myself …

**Comment by steveweenie** [comment]
This is still a problem... i'm trying to add networked splitscreen into my game and its impossible to tell if inputs are split from each other from multiple instances. What even is the solution?
